### ISSUE TITLE
FRED Mnemonic Input Option

### ISSUE THREAD

**Issue by crstnn** [crstnn/BeveridgeNelsonFilter#8]
Add option to main display page

**Comment by jamescmorley** [comment]
Just thinking this through. Will it be possible when entering FRED mnemonic to have it autofill "Time Series Start Date" and "Data Frequency" based on what is on FRED?

**Comment by crstnn** [comment]
Yes we can do that, but does that mean we would allow users exceed the current set of values (Yearly, Quarterly, Monthly, Weekly) considering FRED has a far wider range of frequencies?
